kratos/geometries/brep_surface.h

```diff
@@ -594,6 +611,9 @@ class BrepSurface
 
     BrepCurveOnSurfaceArrayType mEmbeddedEdgesArray;
 
+    ModelPart* mpSurrogateModelPartInner = nullptr;
+    ModelPart* mpSurrogateModelPartOuter = nullptr;
```

Comment:
Move the inits to the constructors please. It makes initialization order a lot clearer and save you some debugging time later on.

```suggestion
    ModelPart* mpSurrogateModelPartInner;
    ModelPart* mpSurrogateModelPartOuter;
```